### PR TITLE
Some readme.md improvements

### PR DESCRIPTION
**Pull request by ludenticus**
+11 New OwnCloud apps page
+12 Nextcloud apps page
+24 Added Markdown cheat-sheet link
+24 Added reference to Nextcloud Notes app
+31 Added reference to Nextcloud and deleted link to ownCloud Notes app
+35 Updated link to ownCloud Tasks, deleted Tasks+ and added Nextcloud Tasks link
+37 Deleted some repetitions
+40-42 Added Nextcloud Notes app (MyOwnNotes doesn't work with Nextcloud)
+44 Added Markor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+[Changelog](https://github.com/pbek/QOwnNotes/blob/develop/CHANGELOG.md) |
+[Issues](https://github.com/pbek/QOwnNotes/issues) |
 [Releases](https://github.com/pbek/QOwnNotes/releases) |
 [Shortcuts](https://github.com/pbek/QOwnNotes/blob/develop/SHORTCUTS.md) |
 [Documentation](http://docs.qownnotes.org) |
 [Knowledge base](http://www.qownnotes.org/Knowledge-base) |
 [Screenshots](http://www.qownnotes.org) |
-[ownCloud Apps page](https://apps.owncloud.com/content/show.php?content=168497) |
+[ownCloud Apps](https://marketplace.owncloud.com/) |
+[Nextcloud Apps](https://apps.nextcloud.com/) |
```
Comment (pbek) [inline README.md]: why did you add the apps pages of oC and Nextcloud

File: README.md [hunk 2]
```diff
@@ -20,28 +21,27 @@
...
 Of course other software, like [Syncthing](https://syncthing.net) or [Dropbox](https://www.dropbox.com) can be used too.
 
-I like the concept of having notes accessible in plain text files, like it is done in the [**ownCloud notes app**](https://github.com/owncloud/notes), to gain a maximum of **freedom**, but I was not able to find a decent desktop note taking tool or a text editor, that handles them well. Out of this need **QOwnNotes** was born.
+I like the concept of having notes accessible in plain text files, like it is done in the ownCloud / Nextcloud notes apps, to gain a maximum of **freedom**, but I was not able to find a decent desktop note taking tool or a text editor, that handles them well. Out of this need **QOwnNotes** was born.
 
 Visit the project page here: [QOwnNotes project page](http://www.qownnotes.org)
 
-To manage your **todo lists** you need [ownCloud tasks](https://apps.owncloud.com/content/show.php/Tasks?content=164356) or [Tasks++](https://apps.owncloud.com/content/show.php/Tasks+Plus?content=170561) as backend, with which you also can manage your todo lists in the **web** and on your mobile devices.
```
Comment (pbek) [inline README.md]: Is Tasks++ dead? QOwnNotes still support it.

File: README.md (hunk 1 above)
Comment (ludenticus) [reply README.md]: I added the Nc apps page because you had already the oC apps page: after updating the oC apps link (`marketplace` insead of `apps` in `owncloud.com`), I found missing a link to the Nc apps page.

File: README.md (hunk 2 above)
Comment (pbek) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (pbek) [reply README.md]: My link was a link to the app page of QOwnNotesAPI, I guess it would make more sense to add links to those, see https://github.com/pbek/qownnotesapi/blob/develop/README.md

Comment (ludenticus) [reply README.md]: You're right, it makes more sense

Comment (pbek) [reply README.md]: Do you care to add the links? :)

**Comment by ludenticus** [comment]
I've just added the oC/Nc API links.

I changed as well `ownCloud apps` and  `Nextcloud apps` to `ownCloud API` and `Nextcloud API`

**Comment by pbek** [comment]
great, thank you!
